Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -0,0 +1,1658 @@
+    proxy = params.get('proxy', False)
+
+    username = params['credentials']['identifier']
+    password = params['credentials']['password']
+
+    return_v1_output = params.get('output_old_format', False)
+
+    command = demisto.command()
+    demisto.debug(f'Command being called is {command}')
+
+    try:
+        urllib3.disable_warnings()
+        client: Client = Client(url, verify_certificate, proxy, headers=HEADERS, username=username, password=password)
+        commands_without_args = {
+            'prisma-cloud-alert-filter-list': alert_filter_list_command,
+
+            'prisma-cloud-trigger-scan': trigger_scan_command,
+        }
+        commands_with_args = {
+            'prisma-cloud-alert-search': alert_search_command,
+            'prisma-cloud-alert-get-details': alert_get_details_command,
+            'prisma-cloud-alert-dismiss': alert_dismiss_command,
+            'prisma-cloud-alert-reopen': alert_reopen_command,
+            'prisma-cloud-remediation-command-list': remediation_command_list_command,
+            'prisma-cloud-alert-remediate': alert_remediate_command,
+
+            'prisma-cloud-config-search': config_search_command,
+            'prisma-cloud-event-search': event_search_command,
+            'prisma-cloud-network-search': network_search_command,
+
+            'prisma-cloud-error-file-list': error_file_list_command,
+
+            'prisma-cloud-resource-get': resource_get_command,
+            'prisma-cloud-account-list': account_list_command,
+            'prisma-cloud-account-status-get': account_status_get_command,
+            'prisma-cloud-account-owner-list': account_owner_list_command,
+
+            'prisma-cloud-host-finding-list': host_finding_list_command,
+            'prisma-cloud-permission-list': permission_list_command,
+        }
+
+        commands_v1 = {
+            'redlock-search-alerts': alert_search_v1_command,
+            'redlock-get-alert-details': alert_get_details_v1_command,
+            'redlock-dismiss-alerts': alert_dismiss_v1_command,
+            'redlock-reopen-alerts': alert_reopen_v1_command,
+            'redlock-get-remediation-details': remediation_command_list_v1_command,
+            'redlock-get-rql-response': rql_config_search_v1_command,
+            'redlock-search-config': config_search_v1_command,
+            'redlock-search-event': event_search_v1_command,
+            'redlock-search-network': network_search_v1_command,
+            'redlock-list-alert-filters': alert_filter_list_v1_command,
+        }
+
+        if command in commands_without_args:
+            return_results(commands_without_args[command](client))
+        elif command in commands_with_args:
+            return_results(commands_with_args[command](client, args))
+        elif command in commands_v1:
+            return_results(commands_v1[command](client, args, return_v1_output))
+        elif command == 'test-module':
+            return_results(test_module(client, params))
```

Comment:
please keep a `result` variable and only use `return_result` once

Reply:
I don't think this is something we do most of the times (it makes the code a bit messy in my opinion).
Maybe you confused with not using `demisto.results()`?